drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -358,10 +379,15 @@ def studio_view(self, context):
         css_urls = (
             'public/css/drag_and_drop_edit.css',
         )
-        js_urls = (
+        js_urls = [
```

Comment:
DRY with [L#258-L#293](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/172/files#diff-81cf2d60a06d8f43d0df802d9aad9417R285). Can we put this in a helper, and then the only thing that's gonna left here is append `public/js/vendor/handlebars-v1.1.2.js` to the urls or even handling it in that helper.

Reply:
☝️